include/SFML/Graphics/Text.hpp

```diff
@@ -224,14 +270,47 @@ class SFML_GRAPHICS_API Text : public Drawable, public Transformable
     Uint32 getStyle() const;
 
     ////////////////////////////////////////////////////////////
-    /// \brief Get the global color of the text
+    /// \brief Get the fill color of the text
+    ///
+    /// \return Fill color of the text
+    ///
+    /// \see setFillColor
+    ///
+    /// \deprecated This function is deprecated and may be removed in future releases.
+    /// Use getFillColor instead.
```

Comment:
Minor remark: the line "This function is deprecated and may be removed in future releases." does not add any information to the `\deprecated` command, I'd rather say something about the backgrounds, i.e. that there are now 2 different colors to set -- fill and outline.

Also, I'd use `getFillColor()` with parentheses at the end for consistency (and I'm not sure if auto-link generation works otherwise).